language-support/qute/qute.tmLanguage.json

```diff
@@ -245,6 +248,35 @@
         }
       ]
     },
+    "section_start_case_is_tag": {
+      "begin": "({)(#)((case|is)\\b)",
+      "end": "(})",
+      "beginCaptures": {
+        "1": {
+          "name": "support.constant.qute"
+        },
+        "2": {
+          "name": "keyword.control"
+        },
+        "3": {
+          "name": "keyword.control"
+        }
+      },
+      "endCaptures": {
+        "1": {
+          "name": "support.constant.qute"
+        }
+      },
+      "patterns": [
+        {
+          "match": "(?<={(#case|#is)\\s)(gt|ge|lt|le|not|ne|in|ni|!in|>|>=|<|<=|!=)(?=\\s\\w+)",
```

Comment:
It might be the case that the regex engine doesn't support using `|` in a look behind. I get an error about this when I tested the regex in https://regex101.com/.

You can try something like this maybe? `(?:(?<={(#case)\s)|(?<={(#is)\s))` I got this from [this StackOverflow suggestion](https://stackoverflow.com/questions/25563891/variable-length-look-behind)